x-pack/test/apm_api_integration/tests/service_overview/instance_details.spec.ts

```diff
@@ -47,11 +47,11 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   );
 
   // FLAKY: https://github.com/elastic/kibana/issues/120056
```

Comment:
Let's remove this one too if it's no longer the case, since the issue is from 2 years ago